front-end/src/Pages/Profile/index.jsx

```diff
@@ -69,18 +71,19 @@ const Profile = () => {
         <label className="change-user-label" htmlFor="name">
           <h2 className="change-user-label-text">Name</h2>
           <input
-            data-testid="profile-name-input"
+            data-testid={role == 'administrador' ? "profile-name" : "profile-name-input"}
```

Comment:
No back end vem como `administrator`, é parecido mas esse `t` importa
```suggestion
            data-testid={role == 'administrator' ? "profile-name" : "profile-name-input"}
```